capstone-backend/src/test/java/com/google/sps/data/BookListTest.java

```diff
@@ -52,72 +55,89 @@ public void testContructors() {
+    assertEquals(GBOOK_IDs_2, bookList2.getGbookIDs());
+    assertTrue(bookList2.getGbookIDs().containsAll(GBOOK_IDs_2));
+    assertTrue(bookList2.getCollaboratorsIDs().isEmpty());
+    assertTrue(bookList2.getId().equals(""));
   }
 
   @Test
   public void testSetters() {
 
     bookList2.setUserID(USER_ID_1);
-    bookList2.setBookshelf(BOOKSHELF_ID_1);
+    bookList2.setGbookIDs(GBOOK_IDs_1);
     bookList2.setCollaboratorsIDs(COLLABORATORS_IDs_2);
     bookList2.setID(ID_2);
 
-    assertEquals(BOOKSHELF_ID_1, bookList2.getBookshelfID());
-    assertEquals(USER_ID_1, bookList2.getUserID());
+    assertTrue(bookList2.getUserID().equals(USER_ID_1));
+    assertEquals(GBOOK_IDs_1, bookList2.getGbookIDs());
+    assertTrue(bookList2.getGbookIDs().containsAll(GBOOK_IDs_1));
     assertEquals(COLLABORATORS_IDs_2, bookList2.getCollaboratorsIDs());
-    assertEquals(ID_2, bookList2.getId());
+    assertTrue(bookList2.getCollaboratorsIDs().containsAll(COLLABORATORS_IDs_2));
+    assertTrue(bookList2.getId().equals(ID_2));
   }
 
   @Test
-  public void testAddRemoveContainsCollaborators(){
-
-    bookList1.addCollaborator(COLLABORATOR_ID_2);
-    assertEquals(true, bookList1.containsCollaborator(COLLABORATOR_ID_2));
-
-    bookList2.addCollaborator(COLLABORATOR_ID_1);
-    assertEquals(true, bookList2.containsCollaborator(COLLABORATOR_ID_1));
-
-    bookList1.removeCollaborator(COLLABORATOR_ID_2);
-    assertEquals(false, bookList1.containsCollaborator(COLLABORATOR_ID_2));
-
-    bookList2.removeCollaborator(COLLABORATOR_ID_1);
-    assertEquals(false, bookList2.containsCollaborator(COLLABORATOR_ID_1));
-
+  public void testAddRemoveContains() {
+
+    bookList1.addGbook(GBOOK_1);
+    bookList1.addCollaborator(COLLABORATOR_ID_1);
+    assertTrue(bookList1.containsGbook(GBOOK_1));
+    assertTrue(bookList1.containsCollaborator(COLLABORATOR_ID_1));
+
+    bookList2.addGbook(GBOOK_2);
+    bookList2.addCollaborator(COLLABORATOR_ID_2);
+    assertTrue(bookList2.containsGbook(GBOOK_2));
+    assertTrue(bookList2.containsCollaborator(COLLABORATOR_ID_2));
+
+    bookList1.removeGbook(GBOOK_1);
+    bookList1.removeCollaborator(COLLABORATOR_ID_1);
+    assertFalse(bookList1.containsGbook(GBOOK_1));
+    assertFalse(bookList1.containsCollaborator(COLLABORATOR_ID_1));
+
+    bookList2.removeGbook(GBOOK_2);
+    bookList2.removeCollaborator(COLLABORATOR_ID_2);
+    assertFalse(bookList2.containsGbook(GBOOK_2));
+    assertFalse(bookList2.containsCollaborator(COLLABORATOR_ID_2));
   }
 
   @Test
   public void testIsEmptyAndClear() {
 
-    assertEquals(false, bookList1.isEmpty());
+    assertFalse(bookList1.isEmpty());
 
     bookList1.clear();
 
-    assertEquals(true, bookList1.isEmpty());
-    assertNull(bookList1.getUserID());
-    assertEquals("", bookList1.getBookshelfID());
-    assertEquals(true, bookList1.getCollaboratorsIDs().isEmpty());
-
+    assertTrue(bookList1.isEmpty());
+    assertTrue(bookList1.getUserID().equals(""));
```

Comment:
Hey Mauro, if you replace your instances of `equals` inside your assertTrue tests with assertEquals I think it will look cleaner and provide better error messages when it fails!
Let me know if you have other questions!

For example,
#138 `assertEquals(bookList1.getUserID(), "")`